grakn-core/src/main/java/ai/grakn/GraknConfigKey.java

```diff
@@ -72,6 +72,8 @@
 
     public static final GraknConfigKey<Path> STATIC_FILES_PATH = key("server.static-file-dir", required(Paths::get));
 
+    public static final GraknConfigKey<Integer> SESSION_CACHE_TIMEOUT = key("knowledge-base.schema-cache-timeout-ms", INT);
```

Comment:
I would keep `_MS` in the name here (or literally make it return a `Duration`, but that's more complicated)